.github/pull_request_template.md

```diff
@@ -0,0 +1,40 @@
+**Purpose of PR?**:
+
+Fixes #
+
+**Does this PR introduce a breaking change?**
+
+**If the changes in this PR are manually verified, list down the scenarios covered:**:
+
+**Additional information for reviewer?** :
+_Mention if this PR is part of any design or a continuation of previous PRs_
+
+
+**Checklist:**
+- [ ] Bug fix. Fixes #<issue number>
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+- [ ] PR Title follows the convention of  `<type>(<scope>): <subject>`
+- [ ] Commit has unit tests
+- [ ] Commit has integration tests
+
+**PLEASE REMOVE BELOW INFORMATION BEFORE SUBMITTING**
```

Comment:
Just put it inside comments.
<!--
Your deets
-->

```md
<!--
Your deets
-->
```
![image](https://user-images.githubusercontent.com/47106543/199009547-0e2dd08b-9cca-482f-8661-711d5e7f4c91.png)